src/services/flutterwave/client.ts

```
@@ -56,14 +66,53 @@ export class FlutterwaveClient implements FintechProvider {
+      while (attempt <= retries) {
+        try {
+          const result = await requestFn();
+          // 2. Record success if the call completes successfully
+          this.breaker.recordSuccess();
+          return result;
+        } catch (error: any) {
+          attempt++;
+          // Only retry on temporary network errors or 5xx server issues; fail immediately on bad data/4xx
+          const isNetworkError = !error.response;
+          const isServerError = error.response?.status >= 500;
+          
```

ESLint

[error] Delete `··········`

(prettier/prettier)